### ISSUE TITLE
Consider moving "option" tables into the models they reference

### ISSUE THREAD

**Issue by stvnrlly** [GSA/notifications-api#79]

**Comment by ccostino** [comment]
Yeah, these are likely in place as a result of some extensive normalization and/or to enforce constraints of values within a field (e.g., a choice or type field can only be one of the records defined in the choice/type table).  However, if there's no other metadata associated with those values other than an ID, there's no real need for them to be in the database and we can handle this within the app code itself (in the models, specifically).

Usually these would be set up as constants (["constants" aren't a real thing in Python, I know](https://realpython.com/python-constants/#defining-your-own-constants-in-python)) that are defined directly in an [`Enum`](https://docs.python.org/3/library/enum.html#enum.Enum) or referenced in a [`tuple`](https://docs.python.org/3.9/tutorial/datastructures.html#tuples-and-sequences) to be used as the structure to define the set of options while also providing you the ability to reference or use the individual values separately.  There are other approaches as well - [data classes](https://docs.python.org/3.9/library/dataclasses.html), [NamedTuples](https://docs.python.org/3.9/library/typing.html#typing.NamedTuple), etc. - but I think going with [`Enum`](https://docs.python.org/3/howto/enum.html) is usually the simplest and most straightforward way.

@xlorepdarkhelm recently did this with the [additional properties on the org model](https://github.com/GSA/notifications-api/pull/621), as an example!
